internal/hardware/validator.go

```diff
@@ -66,6 +66,7 @@ type ValidatorCfg struct {
 	VersionedRequirements         VersionedRequirementsDecoder `envconfig:"HW_VALIDATOR_REQUIREMENTS" default:"[]"`
 	MaxHostDisconnectionTime      time.Duration                `envconfig:"HOST_MAX_DISCONNECTION_TIME" default:"3m"`
 	AgentDockerImage              string                       `envconfig:"AGENT_DOCKER_IMAGE" default:"quay.io/edge-infrastructure/assisted-installer-agent:latest"`
+	EdgeWorkerProductNames        string                       `envconfig:"EDGE_WORKERS_PRODUCT_NAMES" default:"BlueField SoC"`
```

Comment:
https://pkg.go.dev/github.com/vrischmann/envconfig#readme-slices-or-arrays

Reply:
actually it making it less convenient to use, in that case removing all whitespace and making it lowercase (just for consistency)will require joining it back and splitting one more time. 
If you don't mind i would prefer to left it that way